js/renovation/scheduler/workspaces/base/table.tsx

```diff
@@ -0,0 +1,29 @@
+import {
+  Component, ComponentBindings, JSXComponent, OneWay, Slot,
+} from 'devextreme-generator/component_declaration/common';
+
+export const viewFunction = (viewModel: Table) => (
+  <table
+    // eslint-disable-next-line react/jsx-props-no-spreading
+    {...viewModel.restAttributes}
+    className={`${viewModel.props.className}`}
```

Comment:
Why not just `className={viewModel.props.className}`?

Reply:
Sure, I've corrected this.